.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -1,28 +1,28 @@
 name: Feature Request
-description: Suggest an idea for Ferdi
+description: Suggest an idea for Ferdium
 title: ""
 labels: ["enhancement :sparkles:"]
 body:
 - type: markdown
   attributes:
     value: |
-      1. Please remember that if you are logging a bug for some service that has *stopped working*, please log the bug [here](https://github.com/getferdi/recipes/issues)
-      2. If you are requesting support for a new service in Ferdi, please log it [here](https://github.com/getferdi/recipes/pulls)
-      3. Please remember to read the [self-help documentation](https://github.com/getferdi/ferdi#troubleshooting-recipes-self-help) - in case it helps you unblock yourself for issues related to old recipes.
+      1. Please remember that if you are logging a bug for some service that has *stopped working*, please log the bug [here](https://github.com/ferdium/recipes/issues)
+      2. If you are requesting support for a new service in Ferdium, please log it [here](https://github.com/ferdium/recipes/pulls)
```

Should be
```suggestion
      2. If you are requesting support for a new service in Ferdium, please log it [here](https://github.com/ferdium/ferdium-recipes/pulls)
```
(there are other occurence of this link)